tests/integrations/client/http_client_test.go

```diff
@@ -520,14 +526,14 @@ func (suite *httpClientTestSuite) checkScheduleConfig(mode mode, client pd.Clien
 
 	config, err := client.GetScheduleConfig(env.ctx)
 	re.NoError(err)
-	re.Equal(float64(4), config["leader-schedule-limit"])
+	re.Equal(float64(4), config["hot-region-schedule-limit"])
```

Comment:
Why change the scheduler?

Reply:
To avoid the effects of other tests, such as testConfig above.